### DISCUSSION TITLE
Error during processing

### DISCUSSION THREAD

**Discussion by parsoli** [nullpainter/sanchez#92] [General]
Hello!  I am using the Vitality process script for Sanchez.sh.

[2022-12-21 16:17:04] Creating GOES16_FD_sanchez_20221120T180020Z.jpg...
./Sanchez.sh: line 105: -q: command not found

From the Sanchez.sh script I see the below block.  The last line is what is referenced as Line 105.  It's stating 'command not found'.  Do we know what this -q operator is supposed to be doing?

#Build Underlay
		rm /tmp/underlay.jpg > /dev/null 2>&1
		satelliteMonth=$(date -u -d "$thisDate ${thisTime:0:2}:${thisTime:2:2}:${thisTime:4:2} + 5 hour" +"%m")
		xplanet -config "$(dirname "$(readlink -fm "$0")")/Resources/xplanet-$satelliteMonth" -body earth -projection rectangular -num_times 1 -geometry 21600x10800 -date $thisDate.$thisTime -output /tmp/underlay.jpg
		**$sanchezPath -q -s $srcImg -u /tmp/underlay.jpg -r $((10850/$(identify -format '%w' $srcImg))) -n -o $sanchezDstPath16/$newName**

**Comment by nullpainter** [comment]
-q is quiet output. With this enabled, it doesn't write anything to the console. I suspect your path to Sanchez isn't correct. Can you verify that first?

**Reply by JVital2013** [reply]
FYI this got addressed over here: https://github.com/JVital2013/vitality-goes/discussions/33#discussioncomment-4469146

**Reply by nullpainter** [reply]
Ah great, thanks Jamie!
